.github/workflows/require-labels.yml

```diff
@@ -0,0 +1,13 @@
+name: Pull Request Labels
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v1
```

Comment:
Just curious if there is a particular reason for using v1 vs. the [latest v2](https://github.com/marketplace/actions/require-labels) here?

Reply:
No reason aside from that I more or less copied this workflow from an internal repo that was using v1